src/v2/Apps/Order/Utils/shippingUtils.ts

```diff
@@ -147,6 +147,18 @@ export const getShippingOption = (requestedFulfillmentType?: string) => {
   return result
 }
 
+export const getDefaultShippingQuoteId = (order: Shipping_order) => {
```

Comment:
Interesting. I believe almost the same thing - if not exactly the same with sorting is done in `ShippingQuotes.tsx`. It's important I believe that the same logic for sorting is used in both places. We probably should refactor to reuse this code rather than repeat it.

Reply:
+ can adjust!